cmd/oras/internal/display/status/text.go

```diff
@@ -311,3 +311,42 @@ func (miuh *TextManifestIndexUpdateHandler) OnIndexPacked(desc ocispec.Descripto
 func (miuh *TextManifestIndexUpdateHandler) OnIndexPushed(indexRef string) error {
 	return miuh.printer.Println(IndexPromptPushed, indexRef)
 }
+
+// TextBlobPushHandler handles text status output for blob push events.
+type TextBlobPushHandler struct {
+	desc    ocispec.Descriptor
+	printer *output.Printer
+}
+
+// NewTextBlobPushHandler returns a new handler for blob push command.
+func NewTextBlobPushHandler(printer *output.Printer, desc ocispec.Descriptor) BlobPushHandler {
+	return &TextBlobPushHandler{
+		desc:    desc,
+		printer: printer,
+	}
+}
+
+// OnBlobPushSkipped implements BlobPushHandler.
+func (mph *TextBlobPushHandler) OnBlobPushSkipped() error {
```

Comment:
nit: Do you mean `tph` (text push handler) or `bph` (blob push handler)?

Reply:
Yeah I probably forgot to rename the variable name after pasting. Renamed to `bph`.